src/main/java/modtrekt/logic/commands/DoneModuleCommand.java

```diff
@@ -17,6 +20,7 @@
 @Parameters(commandDescription = "Marks a module as done.")
 public class DoneModuleCommand extends Command {
     public static final String COMMAND_WORD = "done";
+    private static int totalCredits;
```

Comment:
This shouldn't be static as DoneModuleCommand is not a singleton - if you need values to be observable, check out [`ObservableValue<T>`](https://docs.oracle.com/javase/8/javafx/api/javafx/beans/value/ObservableValue.html).

Same with the `getTotalCredits` and `refresh` functions.